### PR TITLE
Jatin get profile images from website

### PR DESCRIPTION
**Pull request by JatinAgrawal94**
# Description
Screenshot 2024-09-08 at 1 41 52 PM

## Related PRS (if any):
This frontend PR is related to the [#1116](https://github.com/OneCommunityGlobal/HGNRest/pull/1116/) backend PR.

## Main changes explained:
- Added `confirmRemoveModal.js` to `components/UserProfileModal`.
- Added `suggestedProfileModal.js` to `components/UserProfileModal`.
- Made Changes to `UserProfileModal.css` and `UserProfile.jsx` files.
…

## How to test:
1. check into current branch
2. do `npm install` and `npm run start:local` to run this PR locally
3. Clear site data/cache
4. log as any user
5. go to UserProfile Page.
6. If you have a profile on the onecommunity website, you should either have a profile image(if it was empty before) else you should see a `suggested Profile Image` button to display all the images that match your name.

## Screenshots or videos of changes:

https://github.com/user-attachments/assets/b60da6c0-0375-4fc8-9a5a-e2912c29463f

### CODE REVIEW

**Review by Dk-21 — DISMISSED** [review]
PR-2714
@JatinAgrawal94 
I tested the changes in the code.
I Can't see any of My photo or Suggested Profile Image Button to Display the images that match my name.

Suggestion : Also, We have to manually Enter the name otherwise it leads to the blank page. The name retrieval can be optimized to automatic retrieval from the User profile.

Thank You
PR-2714(1)

**Comment by JatinAgrawal94** [comment]
> @JatinAgrawal94 I tested the changes in the code. I Can't see any of My photo or Suggested Profile Image Button to Display the images that match my name.
> 
> Thank You PR-2714(1)

Hello Denish,
Do you have your bio posted on https://www.onecommunityglobal.org/team ? If yes, Only then your profile image would be changed and you will be able to see suggested Images button.
If you have any other questions, Please let me know.

**Review by Ankuriboh — COMMENTED** [review]
Looks good, but I am not sure if the remove avatar feature is needed.

**Review by mrinalini-raghavendran19 — COMMENTED** [review]
Hey Jatin,

I tried testing this PR, with the suggestion from one of your comments asking you to use your name for testing but it doesn't seem to work for me.

Are there any other things I need to look into before testing this PR?

Screenshot 2024-10-07 at 2 30 33 PM

**Review by Rui-Liu-github — COMMENTED** [review]
Hi, I reviewed an PR #2714 @JatinAgrawal94 .  I pulled the latest version to my local environment to check the fix. I'm not sure if others are experiencing the same issue, but the profile page couldn't be loaded, and errors were displayed in the browser. 

https://github.com/user-attachments/assets/1aae2aba-8f85-41a4-88ce-1c109ffd005f

**Review by Rui-Liu-github — CHANGES_REQUESTED** [review]
I reviewed the PR #2714 just now and try again, but still couldn't be loaded for the profile page. 

image

**Comment by samarth9201** [comment]
I reviewd the PR. When trying to change the profile image, it displays the message "Please click on "Save changes" to save the changes you have made.", however I cannot click the button.

Screenshot (111)

File: src/components/UserProfile/UserProfile.jsx [hunk 1]
```diff
@@ -317,6 +337,7 @@ function UserProfile(props) {
     try {
       const response = await axios.get(ENDPOINTS.USER_PROFILE(userId));
       const newUserProfile = response.data;
+      console.log(newUserProfile.profilePic)
```
Comment (sheetalmangate) [inline src/components/UserProfile/UserProfile.jsx:340]: Please remove console.log()

File: src/components/UserProfile/UserProfile.jsx [hunk 2]
```diff
@@ -882,7 +904,31 @@ function UserProfile(props) {
...
               ) : null}
+              
             </div>
+            <div 
+            style={{ display: 'flex', justifyContent: 'center', gap: '10px', marginTop: '20px' }}>
+              {(userProfile?.profilePic!==undefined)?
+                <Button color="danger" onClick={toggleRemoveModal} className="remove-button">
+                  Remove Image</Button>:<></>}
+              {((userProfile?.profilePic==undefined || 
+                userProfile?.profilePic==null ||
```
Comment (sheetalmangate) [inline src/components/UserProfile/UserProfile.jsx:915]: instead of == use ===

**Comment by sheetalmangate** [comment]
Code changes are looking good only having few suggestions.
1) Remove console.log for printing profile image
2) for comparison use === instead of ==

Could not test it locally.

**Review by sheetalmangate — DISMISSED** [review]
Approving the code only need  below modification
1) Removing console.log()

**Comment by humera314** [comment]
I attempted to navigate to the User Profile page but encountered an error that prevented access.

image image

**Review by ajitp4292 — COMMENTED** [review]
User Profile page is not accessible.
Screenshot 2024-11-02 at 3 53 24 PM

**Comment by JatinAgrawal94** [comment]
> I reviewd the PR. When trying to change the profile image, it displays the message "Please click on "Save changes" to save the changes you have made.", however I cannot click the button.
> 
> Screenshot (111)

The save button is present at the bottom of the page.

**Comment by JatinAgrawal94** [comment]
> User Profile page is not accessible. Screenshot 2024-11-02 at 3 53 24 PM

Please clone the repository again and review. I have made significant changes to for that issue.

**Review by Rupa2400 — DISMISSED** [review]
I reviewed PR #2714 from @JatinAgrawal94 and pulled the latest version into my local environment to test the fix. However, when I tried loading the profile page, it wouldn't display, and I ran into errors in the browser. I'm not sure if anyone else is experiencing the same issue.

PR_2714

**Review by audreydieuanh — DISMISSED** [review]
The profile page isn't loading, and the console shows an error: TypeError: undefined is not an object (evaluating 'suggestedProfilePics.map'). I suspect this is happening in the file src/components/UserProfile/UserProfileModal/suggestedProfileModal.js, where the code attempts to use the .map() method on suggestedProfilePics, which is currently undefined.

test3

**Comment by JatinAgrawal94** [comment]
> The profile page isn't loading, and the console shows an error: TypeError: undefined is not an object (evaluating 'suggestedProfilePics.map'). I suspect this is happening in the file src/components/UserProfile/UserProfileModal/suggestedProfileModal.js, where the code attempts to use the .map() method on suggestedProfilePics, which is currently undefined.
> 
> test3 [ test3 ](https://private-user-images.githubusercontent.com/127930337/382514988-66cbc743-037c-4726-861b-b8421d6fea52.gif?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA1OTM0NzEsIm5iZiI6MTczMDU5MzE3MSwicGF0aCI6Ii8xMjc5MzAzMzcvMzgyNTE0OTg4LTY2Y2JjNzQzLTAzN2MtNDcyNi04NjFiLWI4NDIxZDZmZWE1Mi5naWY_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTAzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTEwM1QwMDE5MzFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT01M2YxZTMzYTA5ZmIwYWY2ODEyN2FmNjJjYWFmZTIzZTU4M2ZhZDA0ZWY1ODExMGFiYTBjZmY1Zjk3ZWU2ZjA0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.c0X8zGXjsfbNkDPNM3Qb9JSbOHZNg8ppK_ZyWO8FAMQ) [ ](https://private-user-images.githubusercontent.com/127930337/382514988-66cbc743-037c-4726-861b-b8421d6fea52.gif?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA1OTM0NzEsIm5iZiI6MTczMDU5MzE3MSwicGF0aCI6Ii8xMjc5MzAzMzcvMzgyNTE0OTg4LTY2Y2JjNzQzLTAzN2MtNDcyNi04NjFiLWI4NDIxZDZmZWE1Mi5naWY_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTAzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTEwM1QwMDE5MzFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT01M2YxZTMzYTA5ZmIwYWY2ODEyN2FmNjJjYWFmZTIzZTU4M2ZhZDA0ZWY1ODExMGFiYTBjZmY1Zjk3ZWU2ZjA0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.c0X8zGXjsfbNkDPNM3Qb9JSbOHZNg8ppK_ZyWO8FAMQ)

Can you please upload the complete video for reference , starting from making required changes to the code.

**Comment by audreydieuanh** [comment]
> > The profile page isn't loading, and the console shows an error: TypeError: undefined is not an object (evaluating 'suggestedProfilePics.map'). I suspect this is happening in the file src/components/UserProfile/UserProfileModal/suggestedProfileModal.js, where the code attempts to use the .map() method on suggestedProfilePics, which is currently undefined.
> > test3
> >       
> >         [
> >           
> >             test3
> >           
> >         ](https://private-user-images.githubusercontent.com/127930337/382514988-66cbc743-037c-4726-861b-b8421d6fea52.gif?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA1OTM0NzEsIm5iZiI6MTczMDU5MzE3MSwicGF0aCI6Ii8xMjc5MzAzMzcvMzgyNTE0OTg4LTY2Y2JjNzQzLTAzN2MtNDcyNi04NjFiLWI4NDIxZDZmZWE1Mi5naWY_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTAzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTEwM1QwMDE5MzFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT01M2YxZTMzYTA5ZmIwYWY2ODEyN2FmNjJjYWFmZTIzZTU4M2ZhZDA0ZWY1ODExMGFiYTBjZmY1Zjk3ZWU2ZjA0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.c0X8zGXjsfbNkDPNM3Qb9JSbOHZNg8ppK_ZyWO8FAMQ)
> >         
> >         
> >           
> >             
> >               
> >             
> >             
> >               
> >               
> >             
> >           
> >           [
> >             
> >               
> >             
> >           ](https://private-user-images.githubusercontent.com/127930337/382514988-66cbc743-037c-4726-861b-b8421d6fea52.gif?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA1OTM0NzEsIm5iZiI6MTczMDU5MzE3MSwicGF0aCI6Ii8xMjc5MzAzMzcvMzgyNTE0OTg4LTY2Y2JjNzQzLTAzN2MtNDcyNi04NjFiLWI4NDIxZDZmZWE1Mi5naWY_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTAzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTEwM1QwMDE5MzFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT01M2YxZTMzYTA5ZmIwYWY2ODEyN2FmNjJjYWFmZTIzZTU4M2ZhZDA0ZWY1ODExMGFiYTBjZmY1Zjk3ZWU2ZjA0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.c0X8zGXjsfbNkDPNM3Qb9JSbOHZNg8ppK_ZyWO8FAMQ)
> >         
> >        [ test3 ](https://private-user-images.githubusercontent.com/127930337/382514988-66cbc743-037c-4726-861b-b8421d6fea52.gif?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA1OTM0NzEsIm5iZiI6MTczMDU5MzE3MSwicGF0aCI6Ii8xMjc5MzAzMzcvMzgyNTE0OTg4LTY2Y2JjNzQzLTAzN2MtNDcyNi04NjFiLWI4NDIxZDZmZWE1Mi5naWY_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTAzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTEwM1QwMDE5MzFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT01M2YxZTMzYTA5ZmIwYWY2ODEyN2FmNjJjYWFmZTIzZTU4M2ZhZDA0ZWY1ODExMGFiYTBjZmY1Zjk3ZWU2ZjA0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.c0X8zGXjsfbNkDPNM3Qb9JSbOHZNg8ppK_ZyWO8FAMQ)
> >       
> >         [
> >           
> >             test3
> >           
> >         ](https://private-user-images.githubusercontent.com/127930337/382514988-66cbc743-037c-4726-861b-b8421d6fea52.gif?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA1OTM0NzEsIm5iZiI6MTczMDU5MzE3MSwicGF0aCI6Ii8xMjc5MzAzMzcvMzgyNTE0OTg4LTY2Y2JjNzQzLTAzN2MtNDcyNi04NjFiLWI4NDIxZDZmZWE1Mi5naWY_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTAzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTEwM1QwMDE5MzFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT01M2YxZTMzYTA5ZmIwYWY2ODEyN2FmNjJjYWFmZTIzZTU4M2ZhZDA0ZWY1ODExMGFiYTBjZmY1Zjk3ZWU2ZjA0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.c0X8zGXjsfbNkDPNM3Qb9JSbOHZNg8ppK_ZyWO8FAMQ)
> >         
> >         
> >           
> >             
> >               
> >             
> >             
> >               
> >               
> >             
> >           
> >           [
> >             
> >               
> >             
> >           ](https://private-user-images.githubusercontent.com/127930337/382514988-66cbc743-037c-4726-861b-b8421d6fea52.gif?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA1OTM0NzEsIm5iZiI6MTczMDU5MzE3MSwicGF0aCI6Ii8xMjc5MzAzMzcvMzgyNTE0OTg4LTY2Y2JjNzQzLTAzN2MtNDcyNi04NjFiLWI4NDIxZDZmZWE1Mi5naWY_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTAzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTEwM1QwMDE5MzFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT01M2YxZTMzYTA5ZmIwYWY2ODEyN2FmNjJjYWFmZTIzZTU4M2ZhZDA0ZWY1ODExMGFiYTBjZmY1Zjk3ZWU2ZjA0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.c0X8zGXjsfbNkDPNM3Qb9JSbOHZNg8ppK_ZyWO8FAMQ)
> >         
> >        [ ](https://private-user-images.githubusercontent.com/127930337/382514988-66cbc743-037c-4726-861b-b8421d6fea52.gif?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA1OTM0NzEsIm5iZiI6MTczMDU5MzE3MSwicGF0aCI6Ii8xMjc5MzAzMzcvMzgyNTE0OTg4LTY2Y2JjNzQzLTAzN2MtNDcyNi04NjFiLWI4NDIxZDZmZWE1Mi5naWY_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTAzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTEwM1QwMDE5MzFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT01M2YxZTMzYTA5ZmIwYWY2ODEyN2FmNjJjYWFmZTIzZTU4M2ZhZDA0ZWY1ODExMGFiYTBjZmY1Zjk3ZWU2ZjA0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.c0X8zGXjsfbNkDPNM3Qb9JSbOHZNg8ppK_ZyWO8FAMQ)
> 
> Can you please upload the complete video for reference , starting from making required changes to the code.

Hi, I'm currently only a member of the PR Review team, so I think I'm not allowed to change code yet. The above is just my suggestion/speculation based on the error.

**Review by Shreyav2000 — CHANGES_REQUESTED** [review]
Hi Jatin,

I reviewed this PR along with the related backend PR and followed the instructions you provided. I updated the server.js file by adding the following lines:

```
[const { getProfileImagesFromWebsite } = require('./helpers/userHelper');
getProfileImagesFromWebsite();]

```

After restarting the server and reloading the page (while testing by changing the name to yours), I couldn’t see any image. I’ve attached a video for your reference.

Additionally, the suggested image feature doesn’t seem to fetch relevant suggestions based on the name entered. Instead, it displays the same three images regardless of the name input.

Refer to the video below:
https://github.com/user-attachments/assets/75a81b13-7d38-489c-90ea-38212f48fbf3

**Review by nishitag5 — CHANGES_REQUESTED** [review]
Hi Jatin,

I reviewed this PR along with the related backend PR and followed your instructions and recordring. After restarting the server and reloading the page (while testing by changing the from one community teams page), the image is not fetched from the website immediately but I can see the changes reflecting after some time.
As it is a corn job. Is there any schedule for the job to run due to which it is delayed?
I have attached the recording and screenshots below:



https://github.com/user-attachments/assets/37bbb58a-898c-4128-a8c9-a97dd1114160
image

**Review by nishitag5 — APPROVED** [review]
HI Jatin,
After taking your changes and following the steps mentioned, I changed the name in the user profile to match with the team member's name available on the One Community website. Then I restarted the server and waited for 5-7 minutes to refresh the profile page. On refreshing the page, I am able to see the profile image based on the user name given in the profile page. 
The changes are working as expected. Please find the recording below:

https://github.com/user-attachments/assets/60b5623c-5ba1-4aee-acbf-6b8551d5225c

**Review by Shreyav2000 — APPROVED** [review]
Hi Jatin,

I reviewed your PR again and I'm ready to approve these changes, as everything is working correctly now. Initially, I thought the changes (particularly the profile image being fetched from the One Community website) would reflect immediately, but I see now that it takes a couple of minutes.

**Review by Sheehan20 — COMMENTED** [review]
Hi @JatinAgrawal94, great work on implementing the profile image fetching feature! Here are my observations:

1. Feature Highlights:
- Added ability to fetch profile images from the One Community website
- Implemented modal for suggested profile images
- Created confirmation modal for image removal
- Enhanced user profile image management

2. Key Components Added:
- `suggestedProfileModal.js`: Modal for displaying suggested profile images
- `confirmRemoveModal.js`: Modal for confirming profile image removal
- Updated `UserProfile.jsx` to support new image selection and removal workflows

3. Notable Technical Implementations:
- Added endpoints for profile image management
- Implemented image conversion to base64
- Added toast notifications for image updates
- Graceful handling of profile image scenarios

4. Observations & Suggestions:
- The feature seems to have a slight delay in image fetching (as noted by reviewers)
- Recommended adding error handling for image conversion
- Consider adding loading states during image fetch/update

5. Testing Recommendations:
- Verify image fetching works across different user profiles
- Test edge cases like no suggested images
- Ensure smooth cross-browser compatibility

Overall, this is a solid implementation that enhances user profile customization. Great job on the detailed implementation!

**Review by honglinchen0524 — COMMENTED** [review]
The profile page isn't loading for some reason. Here is the video (including the code I added following the backend PR instructions):

https://github.com/user-attachments/assets/bc9ab55d-2361-4770-b80a-0b4573cea596

**Comment by jeyanthi-sm** [comment]
Logged in and navigated to 'View Profile'.
Received the following warning:
'It was not possible to retrieve the team codes. Please try again by clicking the icon inside the input auto-complete.'
This was followed by a blank screen.

See the attached video for more details.

https://github.com/user-attachments/assets/d4e4c50f-cfc7-4fdd-9d45-8ab47ba94bd2

**Comment by JatinAgrawal94** [comment]
> The profile page isn't loading for some reason. Here is the video (including the code I added following the backend PR instructions):
> 
>  PR2714.mp4

The user profile page, has been working fine for me. I have tried doing that in firefox, Chrome, Safari. 
Please try again and let me know if the issue still persists.

**Comment by JatinAgrawal94** [comment]
> Logged in and navigated to 'View Profile'. Received the following warning: 'It was not possible to retrieve the team codes. Please try again by clicking the icon inside the input auto-complete.' This was followed by a blank screen.
> 
> See the attached video for more details.
> 
>  PR-2714.mp4

The userprofile screen seems to be working fine for me. I have tried that on chrome, safari and firefox. Please try again and let me know if the issue still persists.

**Comment by one-community** [comment]
Thank you all, merging!
